install/helm/openchoreo-observability-plane/values.yaml

```diff
@@ -8,6 +8,36 @@ global:
   # minimal: Single replica OpenSearch instance will be deployed
   installationMode: minimal
 
+data-prepper:
```

Comment:
Data Prepper config LGTM. Shouldn't we use camel case here by adding an alias to make it consistent with other configs?